packages/bindgen/src/templates/node-wrapper.ts

```diff
@@ -94,21 +95,27 @@ export function generate({ spec: rawSpec, file }: TemplateContext): void {
     // Copied from lib/utils.js.
     // TODO consider importing instead.
     // Might be slightly faster to make dedicated wrapper for 1 and 2 argument forms, but unlikely to be worth it.
-    function _promisify(func) {
+    function _promisify(nullAllowed, func) {
```

Comment:
This change reminds me that it would be great to get this code moved to a separate file some day to get linting and editor support.